packages/contracts/contracts/minter-suite/Minters/MinterDAExpSettlementV3.sol

```diff
@@ -0,0 +1,1101 @@
+                _coreContract
+            ][_projectId]
+        });
+
+        emit ProjectMaxInvocationsLimitUpdated(
+            _projectId,
+            _coreContract,
+            _maxInvocations
+        );
+    }
+
+    /**
+     * @notice Sets auction details for project `_projectId`.
+     * @param _projectId Project ID to set auction details for.
+     * @param _coreContract Core contract address for the given project.
+     * @param _auctionTimestampStart Timestamp at which to start the auction.
+     * @param _priceDecayHalfLifeSeconds The half life with which to decay the
+     *  price (in seconds).
+     * @param _startPrice Price at which to start the auction, in Wei.
+     * @param _basePrice Resting price of the auction, in Wei.
+     * @dev Note that it is intentionally supported here that the configured
+     * price may be explicitly set to `0`.
+     */
+    function setAuctionDetails(
+        uint256 _projectId,
+        address _coreContract,
+        uint64 _auctionTimestampStart,
+        uint64 _priceDecayHalfLifeSeconds,
+        uint128 _startPrice,
+        uint128 _basePrice
+    ) external {
+        AuthLib.onlyArtist({
+            _projectId: _projectId,
+            _coreContract: _coreContract,
+            _sender: msg.sender
+        });
+        // CHECKS
+        DAExpLib.DAProjectConfig
+            storage _auctionProjectConfig = _auctionProjectConfigMapping[
+                _coreContract
+            ][_projectId];
+        SettlementExpLib.SettlementAuctionProjectConfig
+            storage _settlementAuctionProjectConfig = _settlementAuctionProjectConfigMapping[
+                _coreContract
+            ][_projectId];
+
+        // require valid start price on a settlement minter
+        require(
+            SettlementExpLib.isValidStartPrice({
+                _startPrice: _startPrice,
+                _settlementAuctionProjectConfig: _settlementAuctionProjectConfig
+            }),
+            "Only monotonic decreasing price"
+        );
+        // do not allow a base price of zero (to simplify logic on this minter)
+        require(_basePrice > 0, "Base price must be non-zero");
+        // require valid half life for this minter
+        require(
+            (_priceDecayHalfLifeSeconds >= minimumPriceDecayHalfLifeSeconds),
+            "Price decay half life must be greater than min allowable value"
+        );
+
+        // EFFECTS
+        DAExpLib.setAuctionDetailsExp({
+            _DAProjectConfig: _auctionProjectConfig,
+            _auctionTimestampStart: _auctionTimestampStart,
+            _priceDecayHalfLifeSeconds: _priceDecayHalfLifeSeconds,
+            _startPrice: _startPrice,
+            _basePrice: _basePrice,
+            // we set this to false so it prevents artist from altering auction
+            // even after max has been invoked (require explicit auction reset
+            // on settlement minter)
+            _maxHasBeenInvoked: false
```

Comment:
Cc @jakerockland @lyaunzbe on this update to account for upstream changes.
Unlike the non-settlement minters, I do not think this minter should let an auction be re-configured if max invocations have been hit.

Reply:
Updated to the new proposed names based on convo w/ @lyaunzbe